### DISCUSSION TITLE
Is EntityManager meant to be disposable?

### DISCUSSION THREAD

**Discussion by Atlinx** [twinlogix/typetta#205] [Q&A] [answered]
Is it okay to create a new EntityManager instance every time a new request is received?

**Comment by edobrb** [comment, marked as answer]
Absolutely yes, you should create a new EntityManager for each request. In some cases you may want to instantiate an EntityManager with different metadata depending on the user. Also, all the DAO inside the EntityManager are lazy instantiated so it's quite lightweight.
